source/angular-time-picker.js

```diff
@@ -6,47 +6,59 @@ angular.module( "vokal.timePicker", [] )
     {
         "use strict";
 
+        var defaultFormat = "h:mm a";
+        var defaultDateStr = "1/1/1990" + " ";
+
+        function validateTime( time )
+        {
+            return !!time && angular.isString( time ) && !isNaN( new Date( defaultDateStr + time ).getTime() );
+        }
+        function validateDate( date )
+        {
+            return date && angular.isFunction( date.getTime ) && !isNaN( date.getTime() );
+        }
+        function convertToDate( str )
+        {
+            return validateDate( str ) ? str : new Date( str );
+        }
```

Comment:
Not entirely sure why you copied these into two files instead of making a file for both to consume.